hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -16,6 +16,7 @@
 ##  - attr_dw:             Width of wakeup counters
 {
   name:               "pinmux",
+  one_line_desc:      "Multiplexes between on-chip hardware blocks and pins and can be configured at runtime",
```

Comment:
"blocks and pins and can be configured at runtime" -> "blocks and pins, can be configured at runtime"